test/test_mpi_tree.jl

```diff
@@ -189,6 +189,20 @@ CI_ON_WINDOWS = (get(ENV, "GITHUB_ACTIONS", false) == "true") && Sys.iswindows()
         rtol = 0.001)
     end
   end
+
+  @trixi_testset "elixir_euler_density_wave_restart.jl" begin
+    trixi_include(@__MODULE__, joinpath(EXAMPLES_DIR, "elixir_euler_density_wave_extended.jl"));
+    l2_expected, linf_expected = analysis_callback(sol)
+    Trixi.mpi_isroot() && println("═"^100)
+    Trixi.mpi_isroot() && println(joinpath(EXAMPLES_DIR, "elixir_euler_density_wave_restart.jl"))
```

Comment:
Thank you for the review! It's a part of actual test, since I don't call `@test_trixi_include`, the running file name in test won't be printed without these lines and it will be complicated in case of error to identify the problematic elixir. The same was implemented here:
https://github.com/trixi-framework/Trixi.jl/blob/e22b11e5859b5168e24dabb4cb34a6a35fce3e66/test/test_mpi_p4est_2d.jl#L22
Do you think it is unnecessary?

Reply:
OK, thanks for the explanation. No, I think it that case it is a good idea to have it 👍